ironfish/src/network/peers/connections/webRtcConnection.ts

```diff
@@ -44,43 +70,58 @@ export class WebRtcConnection extends Connection {
       this.addLatencyWrapper()
     }
 
-    // TODO: This is using google STUN internally, we need to
-    // make it use any of the websocket peers
-    this.peer = new SimplePeer({ initiator, wrtc })
+    // TODO: Use our own STUN servers
```

Comment:
As far as I can tell, google's stun server is okay for production use.

Reply:
I think the main potential issues were geo-based and reliance on third-party servers.